scripts/docker/test_astrobee.Dockerfile

```diff
@@ -6,7 +6,9 @@ ARG REMOTE=astrobee
 FROM ${REMOTE}/astrobee:latest-ubuntu${UBUNTU_VERSION}
 
 # Run tests
-RUN cd /src/astrobee && catkin build --make-args tests \
-	&& catkin build --make-args test -j1\
+RUN cd /src/astrobee \
+	&& catkin build --no-status --force-color --make-args tests \
+	&& { catkin build --no-status --force-color --make-args test -j1 || true; } \
```

Comment:
why are we running the tests twice with the errors suppressed the first time?

`catkin build --no-status --force-color --make-args test -j1`
already runs the tests

Reply:
It looks like exactly the same, but we need to add the -j1 argument to make sure only one test runs at a time (if there are multiple tests running simultaneously sometimes rosnodes interfere with each other, or multiple gazebo servers try to start)

Reply:
I'm going to run both commands on my local machine and make sure both have the same number of tests :)

Reply:
Ok. The other issue is that, as I understand it, the call in L11 builds formal test result output files that are then collected and evaluated for overall success/failure check in L14. The question I didn't figure out is whether the call in L13 also builds the test result output files in the same way.

Reply:
Another point is that `-j1` might not be needed for testing within Docker if the problem with multiple concurrent tests only arises for integrated tests that are disabled in Docker anyway. Forcing `-j1` could certainly be slowing down the testing, depending on the resources available to the Docker container.

Reply:
We've had this problem with github CI too

Reply:
> Ok. The other issue is that, as I understand it, the call in L11 builds formal test result output files that are then collected and evaluated for overall success/failure check in L14. The question I didn't figure out is whether the call in L13 also builds the test result output files in the same way.

Testing this sort of things is why docker is so useful :)

Reply:
So it seems that the L13 command on it's own doesn't actually run the tests.
But using both will just take double the time because the tests can run after the L11 cmd. I don't think we want all our CI pipelines to take double the time. If getting each test output is necessary, I would put it with a | (or) command such that it only happens in case of failure

Reply:
Should be done in 28c03a22c0b54d0b61fdd411737cfce29a5a6290

The strategy is explained in `run_tests.sh`. I verified it has the desired behavior both when all tests succeed (don't re-run verbosely) and when a test fails (do re-run verbosely). Hopefully the CI testing will behave the same way.

Note: I think actually running the tests (versus compiling FSW and also compiling the tests) may be a pretty small proportion of the overall CI process, so running them twice might not affect the total CI time very much. But it's still good to take the easy performance improvement.